.github/workflows/branch-build.yaml

```
@@ -15,16 +15,89 @@ jobs:
     runs-on: ubuntu-latest
```

I noticed that it will also run on draft PRs. After review I will add:
```suggestion
    runs-on: ubuntu-latest
    if: github.event.pull_request.draft == false
```